src/probeinterface/probe.py

```diff
@@ -838,36 +838,68 @@ def from_dict(d: dict) -> "Probe":
 
     def to_numpy(self, complete: bool = False) -> np.array:
         """
-        Export to a numpy vector (structured array).
-        This vector handles all contact attributes.
+        Export the probe to a numpy structured array.
+        This array handles all contact attributes.
 
-        Equivalent to the 'to_dataframe()' pandas function, but without pandas dependency.
+        Similar to the 'to_dataframe()' pandas function, but without pandas dependency.
 
-        Very useful to export/slice/attach to a recording.
+        The intended use is to attach this array to a recording object as a property ("contact vector")
 
         Parameters
         ----------
         complete : bool, default: False
             If True, export complete information about the probe,
-            including contact_plane_axes/si_units/device_channel_indices
+            including contact_plane_axes/si_units/device_channel_indices.
 
-        returns
-        ---------
+        Returns
+        -------
         arr : numpy.array
-            With complex dtype
+            Structured array with the following dtype schema:
+
+            dtype = [
+                ('x', 'float64'),
+                ('y', 'float64'),
+                ('z', 'float64', optional),
+                ('contact_shapes', 'U64'),
+                ...
+                # Shape parameters
+                ('shape_param_1', 'float64'),
+                ('shape_param_2', 'float64'),
+                ...
+                ('shank_ids', 'U64'),
+                ('contact_ids', 'U64'),
+                # The rest is added if `complete=True`
+                ('device_channel_indices', 'int64', optional),
+                ('si_units', 'U64', optional),
+                ('plane_axis_x_0', 'float64', optional),
+                ('plane_axis_x_1', 'float64', optional),
+                ('plane_axis_y_0', 'float64', optional),
+                ('plane_axis_y_1', 'float64', optional),
+                ('plane_axis_z_0', 'float64', optional),
+                ('plane_axis_z_1', 'float64', optional),
+                ...
```

Comment:
Where are these ... suppose to mean?

Reply:
It means that after that there are variable number of lines.

Reply:
That's what I thought, but wanted to confirm it wasn't something else. Thanks.

Reply:
Maybe there is a better way of saying that? Not sure.

Reply:
I think for people use to matrix notation it makes perfect sense. for newer users I think just explicitly saying;

... variable number of rows

would be easier or them. It will just look less pleasing and might annoy people use to this notation.

Reply:
I think that's better as well.

Reply:
Changed.